integration/ig/non-k8s/integration_test.go

```diff
@@ -39,6 +42,8 @@ func TestMain(m *testing.M) {
 		os.Exit(0)
 	}
 
+	containerFactory = NewContainerFactory(*runtime)
```

Comment:
Should we explicity add check for [supported runtime](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/integration/ig/k8s/integration_test.go#L59) or you want it to panic ?